preinstall.sh

```diff
@@ -215,6 +215,12 @@ if [ -f /etc/debian_version ]; then
       apt-get install -y automake build-essential gobject-introspection gtk-doc-tools libglib2.0-dev libjpeg-dev libpng12-dev libwebp-dev libtiff5-dev libexif-dev libgsf-1-dev liblcms2-dev libxml2-dev swig libmagickcore-dev curl
       install_libvips_from_source
       ;;
+    stretch)
+      # Debian 9
+      echo "Installing libvips dependencies via apt-get"
+      apt-get install -y automake build-essential gobject-introspection gtk-doc-tools libglib2.0-dev libjpeg-dev libpng-dev libwebp-dev libtiff5-dev libexif-dev libgsf-1-dev liblcms2-dev libxml2-dev swig libmagickcore-dev curl
+      install_libvips_from_source
+      ;;
```

Comment:
Is it not this case block redundant?

Reply:
The only difference to Debian 8 is, that `libpng12-dev` is now `libpng-dev`. Apart from that, everything is identical. I wasn't sure, if it's nicer to do it that way or to add `stretch` here

`jessie|trusty|utopic|vivid|wily|xenial|qiana|rebecca|rafaela|freya|rosa|sarah|serena)`

and do explicit checking inside the case block.